src/pybind/mgr/dashboard/HACKING.rst

```diff
@@ -725,6 +725,41 @@ same applies to other request types:
 | DELETE       | Yes        | delete         | 204         |
 +--------------+------------+----------------+-------------+
 
+How to use a custom API endpoint in a RESTController?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you have no access restriction you can use ``@Endpoint`` but if you have you
+use instead ``@RESTController.Collection`` or ``@RESTController.Resource``, if
```

Comment:
```suggestion
and ``RESOURCE_ID`` is set you'll have to use ``@RESTController.Collection`` or ``@RESTController.Resource`` instead.
```

Reply:
You only can use ``@RESTController.Resource`` if ``RESOURCE_ID`` is set, but you can use ``@RESTController.Collection`` without having the ``RESOURCE_ID`` set.